packages/formik-inputs/src/components.tsx

```diff
@@ -70,3 +72,6 @@ export const TextArea: React.FC<TextAreaProps> = props => (
 export const TextInput: React.FC<TextInputProps> = props => (
   <BTextInput {...useField(props)} />
 );
+export const RadioGroup: React.FC<RadioGroupProps> = props => (
+  <BRadioGroup options={props.options} {...useField(props)} />
```

Comment:
You shouldn't need to pass options through explicitly. What was the issue here?

Reply:
hmm, yeah — I was getting TS errors but I think it may have put this in before I realized that I needed to `yarn build` to see my updates reflected. Looks ok without it.